server/public/styles.css

```diff
@@ -253,6 +272,85 @@ img {
   justify-content: center;
 }
 
+.duplicate {
+  background-color: black;
+  box-sizing: border-box;
+  border: 1px solid gray;
+  border-radius: 5px;
+  width: 175px;
+  height: 65px;
+  outline: none;
+  resize: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+.pad-top {
+  padding-top: 10px;
+}
+
+.sign-up-container {
+  background-color: white;
+  box-sizing: border-box;
+  border: 1px solid gray;
+  border-radius: 5px;
+  outline: none;
+  resize: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.sign-up-header {
+  font-family: Inter, sans-serif;
+  margin-bottom: 0;
+}
+
+.sign-up-para {
+  font-family: Inter, sans-serif;
+  font-size: x-small;
+  margin-bottom: 0;
```

Comment:
There is a _lot_ of duplication in your CSS. try to use some element selectors where it makes sense, like all inputs for fonts.

And for other elements, let them inherit the font. Box sizing border-box should be in your reset.css for * universal selector. Please try to combine a lot of your CSS